feat: added Kube-green module README (#9)
mia-platform/distribution: modules/scheduling/kube-green/base/README.md

# Traefik Module

> Kube-green is a Kubernetes addon that allows you to shut down the workload you don't need with the definition of *schedules*.

Reviewer: ```suggestion
Kube-green is a Kubernetes addon that allows you to shut down the workloads you don't need with the definition of *schedules*.
```

> [Here is the link to Kube-green official repository.]

Reviewer: I think that here a link is missing 😄

Author: added the doc link

## What the base module contains

- **CRDs:** Kube-green custom resource definitions

> - **Resources (Traefik ingress):**

Reviewer: ```suggestion
- **Resources:**
```

typo from copy and paste

>   - **Configs:** configurations of related resources.

Reviewer: Better be explicit that we deploy a validation webhook for the kube-green resources

>   - **Workload:** deployments and services for Kube-green ingress and its dashboards

Reviewer: ```suggestion
  - **Workload:** deployments and services for Kube-green controller and its service
```

  - **RBAC:** The necessary service account, `ClusterRole` and `ClusterRoleBinding`.

## Flavors

This module has no flavors

## Compatibility Matrix

| Module Version | Tool Version   |
|----------------|----------------|

> | 1.24.0         | 0.5.0          |

Reviewer: ```suggestion
| 1.24.0         | 0.4.0          |
```

We haven't already update to 0.5.0
